src/main/java/Duke.java

```diff
@@ -1,10 +1,70 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        /*
+         * Duke greetings
+         */
+        String logo = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello! I'm Duke\n    What can I do for you?");
+        System.out.println("    ____________________________________________________________\n");
+
+        /*
+         * Takes in user inputs, and echoes them. If user input == "bye", then program
+         * exits
+         */
+        Scanner userInput = new Scanner(System.in);
+        String currentInput = userInput.nextLine();
+        // while (!"bye".equals(currentInput)) {
+        // System.out.println("
+        // ____________________________________________________________");
+        // System.out.println(" " + currentInput);
+        // System.out.println("
+        // ____________________________________________________________\n");
+        // currentInput = userInput.nextLine();
+        // }
+
+        TaskManager t = new TaskManager();
+        while (!"bye".equals(currentInput)) {
```

Comment:
It would be good if you make the conditions more understandable such as creating a new variable